.github/workflows/release.yml

```diff
@@ -0,0 +1,105 @@
+name: Releases
+
+on:
+  push:
+    tags:
+      - "v[0-9]+.[0-9]+.[0-9]+"
+
+jobs:
+  build-windows:
+    runs-on: windows-latest
+    timeout-minutes: 60
+
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          submodules: true
+      - uses: msys2/setup-msys2@v2
+        with:
+          update: true
+          path-type: inherit
+          install: >-
+            curl
+            make
+            cmake
+
+      - shell: msys2 {0}
+        run: |
+          pacman -S mingw-w64-x86_64-gcc mingw-w64-x86_64-cmake mingw-w64-x86_64-make mingw-w64-x86_64-curl libcurl openssl-devel libcurl-devel autoconf texinfo mingw-w64-x86_64-libtool --noconfirm
+          make
+
+          mkdir ./txiki-windows-x86
+          cp ./build/tjs.exe ./txiki-windows-x86/tjs.exe
+          cp ./build/libtjs.a ./txiki-windows-x86/libtjs.a
```

Comment:
This is not needed for the executable to run. Since we are not packing headers, it's not very usable. Let's leave it out and reconsider later please.